### PR TITLE
add tag view for android

### PR DESCRIPTION
**Pull request by KazuCocoa**
add `-android viewtag` for Espresso Driver #237

### CODE REVIEW

File: appium/webdriver/common/mobileby.py [hunk 1]
```diff
@@ -20,5 +20,6 @@ class MobileBy(By):
     IOS_UIAUTOMATION = '-ios uiautomation'
     IOS_CLASS_CHAIN = '-ios class chain'
     ANDROID_UIAUTOMATOR = '-android uiautomator'
+    ANDROID_TAGVIEW = '-android viewtag'
```
Comment (mykola-mokhnach) [inline appium/webdriver/common/mobileby.py]: ANDROID_VIEWTAG?

File: appium/webdriver/webdriver.py [hunk 2]
```diff
@@ -341,6 +341,30 @@ def find_elements_by_android_uiautomator(self, uia_string):
         """
         return self.find_elements(by=By.ANDROID_UIAUTOMATOR, value=uia_string)
 
+    def find_element_by_android_viewtag(self, tag_string):
```
Comment (mykola-mokhnach) [inline appium/webdriver/webdriver.py]: simply `tag`

File: appium/webdriver/webdriver.py [hunk 3]
```diff
@@ -341,6 +341,30 @@ def find_elements_by_android_uiautomator(self, uia_string):
...
+        """Finds element by tag in Android.
+        https://developer.android.com/reference/android/view/View
+
+        :Args:
+         - tag_string - The element name by tag view in Android
```
Comment (mykola-mokhnach) [inline appium/webdriver/webdriver.py]: The tag name of the view to look for

File: appium/webdriver/webdriver.py [hunk 4]
```diff
@@ -341,6 +341,30 @@ def find_elements_by_android_uiautomator(self, uia_string):
         """
         return self.find_elements(by=By.ANDROID_UIAUTOMATOR, value=uia_string)
 
+    def find_element_by_android_viewtag(self, tag):
+        """Finds element by tag in Android.
```
Comment (mykola-mokhnach) [inline appium/webdriver/webdriver.py]: Perhaps, it worth to mentions that such locator only works for Espresso driver

Comment (KazuCocoa) [reply appium/webdriver/webdriver.py]: :+1: Yeah, I'm appending `find_element_by_android_viewtag` section in README. I'll mention the point here as well.

**Pull request merged**